src/components/Explore/TokenRow.tsx

```diff
@@ -0,0 +1,221 @@
+import CurrencyLogo from 'components/CurrencyLogo'
+import { useCurrency, useToken } from 'hooks/Tokens'
+import { TimePeriod, TokenData } from 'hooks/useTopTokens'
+import React from 'react'
+import { ArrowDownRight, ArrowUpRight, Heart } from 'react-feather'
+import styled from 'styled-components/macro'
+
+const TokenRowWrapper = styled.div`
+  width: 100%;
+  height: 60px;
+  display: grid;
+  padding: 0px 12px;
+  grid-template-columns: 40px 32px 200px 107.5px 107.5px 107.5px 107.5px 172px 62px;
```

Comment:
can you make these 107.5 values 108 so they are divisible by 4?

Reply:
Ya these were the numbers given by Fred, I made them %s for now? Is that okay?

Reply:
Use the effective % splits here per @infredible's slack message